envs/monkey_zoo/terraform/monkey_zoo.tf

```diff
@@ -283,409 +283,433 @@ resource "google_compute_instance_from_template" "rdpgrinder-13" {
 */
 
 resource "google_compute_instance_from_template" "mimikatz-14" {
-  name = "${local.resource_prefix}mimikatz-14"
+  name                     = "${local.resource_prefix}mimikatz-14"
   source_instance_template = local.default_windows
-  boot_disk{
+  boot_disk {
     initialize_params {
       image = data.google_compute_image.mimikatz-14.self_link
     }
     auto_delete = true
   }
   network_interface {
-    subnetwork="${local.resource_prefix}monkeyzoo-main"
-    network_ip="10.2.2.14"
+    subnetwork = "${local.resource_prefix}monkeyzoo-main"
+    network_ip = "10.2.2.14"
     access_config {
-    // Allows Ephemeral IPs
+      // Allows Ephemeral IPs
     }
   }
 }
 
 resource "google_compute_instance_from_template" "mimikatz-15" {
-  name = "${local.resource_prefix}mimikatz-15"
+  name                     = "${local.resource_prefix}mimikatz-15"
   source_instance_template = local.default_windows
-  boot_disk{
+  boot_disk {
     initialize_params {
       image = data.google_compute_image.mimikatz-15.self_link
     }
     auto_delete = true
   }
   network_interface {
-    subnetwork="${local.resource_prefix}monkeyzoo-main"
-    network_ip="10.2.2.15"
+    subnetwork = "${local.resource_prefix}monkeyzoo-main"
+    network_ip = "10.2.2.15"
     access_config {
-    // Allows Ephemeral IPs
+      // Allows Ephemeral IPs
     }
   }
 }
 
 resource "google_compute_instance_from_template" "mssql-16" {
-  name = "${local.resource_prefix}mssql-16"
+  name                     = "${local.resource_prefix}mssql-16"
   source_instance_template = local.default_windows
-  machine_type = "e2-highcpu-4"
-  boot_disk{
+  machine_type             = "e2-highcpu-4"
+  boot_disk {
     initialize_params {
       image = data.google_compute_image.mssql-16.self_link
     }
     auto_delete = true
   }
   network_interface {
-    subnetwork="${local.resource_prefix}monkeyzoo-main"
-    network_ip="10.2.2.16"
+    subnetwork = "${local.resource_prefix}monkeyzoo-main"
+    network_ip = "10.2.2.16"
+    access_config {
+      // Allows Ephemeral IPs
+    }
+  }
+}
+
+resource "google_compute_instance" "snmp-20" {
```

Comment:
We should put this in the `europe-west1-b` zone, as it has much higher quotas than `europe-west3-a`.

Reply:
That sounds good. I can re-generate the instances on `europe-west1`. Our config currently defaults to `europe-west3` -- I'll update it to point to `europe-west1` by default.

Reply:
Will pointing to europe-west1 by default have unintended consequences? The BB tests have a dictionary of regions to instances.

Reply:
The different network segments are to allow for different types of testing. But the different regions are are because, initially, everything was in west3, but we hit the CPU quota, so we started building things in west1. There hasn't been time to migrate everything to west1.